_scicomputing/hutchgo_overview.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Hutch Globus Online - HutchGO
+last_modified_at: 2024-07-01
+primary_reviewers: atombaby
+---
+
+Globus is a non-profit service of the University of Chicago.  It is a combination of software (the Globus server and client tools) and services (the Globus control services) providing a number of different functions useful in research- data transfer, compute, and automation.
+
+HutchGO- Hutch Globus Online- is a deployment of the Globus data transfer services for the Hutch research community. HutchGO can be used for moving data within the Hutch and to make data available to external collaborators.  Via HutchGO you can move data between storage platforms and share data with internal and external collaborators without creating new accounts, changing storage groups, or moving data.
+
+HutchGO is available for the advancement of the Hutch's mission.  There is no charge for Globus services.
+
+## Key Concepts
+
+### The Globus Identity
+
+As part of its services, Globus operates an _identity_ service.  Each individual who uses Globus has a Globus ID.  A Globus ID can be accessed by authenticating using a number of different sources- an institutional login (e.g. our Hutch O365 login), ORCID, Google account, or GitHub login can all be used to authenticate to your Globus ID.  You can have multiple login sources connected to the same Globus ID.
+
+> NOTE: while a single Globus ID can be linked to many different authentication sources, most of the operations described here require you to log into Globus with your HutchNet ID.  Logging into Globus with your ORCID will not get you access to the same resources as logging in with your "@fredhutch.org" institutional account.
+
+### Collections 
+
+Central to the use of Globus is the _collection_- a collection of paths to data on the underlying storage gateway. Data is transferred between collections.  Policies (like who can access data via Globus, how paths are shared, etc) are set on the collection.
+
+There are two kinds of collections- _mapped_ and _guest_.  A mapped collection requires that you have a HutchNet ID and that your HutchNet ID has access to the storage systems underneath Globus.  Your login to Globus is _mapped_ to a HutchNet ID on the storage system.
+
+> For example, if I have logged into Globus using my "mrg@fredhutch.org" institutional account and I access a collection that provides access to Fast file, I will only have the access privileges that my user account has on Fast.  In the mapped collection, Globus provides no "extra" privileges.
```

Comment:
Might not want your full email here

Reply:
Removed in favor of more detailed permission discussion